### PR TITLE
Adding new command - test-content

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added a new hidden command `test-content` that will run the test-content step in the build.

The proxy, ParallelLoggingManager, Docker and locking mechanism were copied as well and will be deleted from content in a seperate PR.

## Related issues: 
https://github.com/demisto/etc/issues/32220#event-4155053781

### CODE REVIEW

File: demisto_sdk/commands/test_content/TestContentClasses.py [hunk 1]
```diff
@@ -0,0 +1,1639 @@
...
+        while 'configurations' not in res:
+            if total_sleep == TIMEOUT:
+                self.logging_module.error(
+                    f"Timeout - failed to get all integration configuration. Error: {res}")
+                return None
+
+            time.sleep(SLEEP_INTERVAL)
+            total_sleep += SLEEP_INTERVAL
+
+        all_configurations = res['configurations']
```
Comment (roysagi) [inline demisto_sdk/commands/test_content/TestContentClasses.py:427]: Seems you have code duplication on this - you might want to move this to a tool.

File: demisto_sdk/commands/test_content/TestContentClasses.py [hunk 2]
```diff
@@ -0,0 +1,1639 @@
...
+            return False
+        if int(res[1]) != 200:
+            self.build_context.logging_module.error(f'delete integration instance failed\nStatus code {res[1]}')
+            self.build_context.logging_module.error(pformat(res))
+            return False
+        if self.integration_instance:
+            self.integration_instance = {}
+        return True
+
+    def test_integration_instance(self, client: DefaultApi) -> bool:
```
Comment (roysagi) [inline demisto_sdk/commands/test_content/TestContentClasses.py:1079]: seemingly this is not used anywhere - where are we testing the instance?

File: demisto_sdk/commands/test_content/TestContentClasses.py [hunk 3]
```diff
@@ -0,0 +1,1639 @@
...
+            is_playback_run:
+            status:
+        """
+        if status == PB_Status.COMPLETED:
+            self.build_context.logging_module.success(f'PASS: {self} succeed')
+            self._add_to_succeeded_playbooks()
+
+        elif status == PB_Status.NOT_SUPPORTED_VERSION:
+            self.build_context.logging_module.info(f'PASS: {self} skipped - not supported version')
+            self._add_to_succeeded_playbooks()
```
Comment (roysagi) [inline demisto_sdk/commands/test_content/TestContentClasses.py:1444]: maybe add to skipped playbooks or remove

**Review by roysagi — APPROVED** [review]
Amazing work here - please see my comments.
Also please add some dev notes to sections of untrivial methods as we spoke - this could be a major difference in debugging this in the future.

**Comment by Itay4** [comment]
build fails on coverage decrease
merging

**Pull request merged**